api/src/modules/h3-data/dto/h3-by-resolution.dto.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * DTO for querying a resolution given a H3 ID
+ */
+import { Type } from 'class-transformer';
+import { IsEnum, IsNumber, IsOptional } from 'class-validator';
+
+/**
+ * Default (and max) resolution to retrieve h3 indexes if none is given
+ */
+export const DEFAULT_RESOLUTION = 6;
```

Comment:
could you explain why we went for the value that (AFAIK) produces the most data? Wouldn't it be more reasonable to a) use the cheapest default, or b) make the res mandatory on the request? (I actually prefer b over a TBH)

Reply:
Because max resolution implies the full map, retrieving the cheapest result will imply returning the max zoom, which I am not sure if it makes sense. 
I totally see your point and I do agree with it, I thought, without asking anyone and thats not ok, making default the full data was the right answer.

Let me check with FE which would the solution will them